docs/rfcs/read-only-replica.md

```diff
@@ -0,0 +1,60 @@
+# Read-only replicas
+
+We want to be able to spin-up read-only compute nodes.
+
+## Scope of this feature
+
+- We want to be able to spin-up multiple read-only replicas.
+- We should be able to spin-up RO node, if primary is inactive.
+
+- We don't want "snapshot" read-only computes at fixed LSN. Users can use branches for that.
+- We don't need replica promotion. For failower we rely on quick primary node restarts.
```

Comment:
The primary goal of using replicas in Postgres is HA, not load balancing.
Yes, Neon allows to quickly restart compute. But with cold cache.
And performance of Neon greatly depends on local caches: if data set fits in compute node then it is comparable with standalone Postgres, if not, then at random-access workload it is  ~10x times slower.

So if we assume that in most cases working data set fit in compute cache, then keeping warm replica seems to be good solution for fast switch-over without cache warming. But it will not be possible if there no read-only workload at replica and we apply WAL records at replica only to available pages.

So I am not sure whether we need replicas for fast HA, but at least I think that we should mention this option and explain related problems.